.github/workflows/build-participation.yml

```
@@ -41,7 +42,7 @@ jobs:
         run: echo "GIT_TAG=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
 
       - name: Set Registry Image Based on Tag
-        if: startsWith(env.GIT_TAG, 'mainnet') || startsWith(env.GIT_TAG, 'betanet') || startsWith(env.GIT_TAG, 'testnet') || startsWith(env.GIT_TAG, 'testnet-v1.0')
+        if: startsWith(env.GIT_TAG, 'mainnet') || startsWith(env.GIT_TAG, 'betanet') || startsWith(env.GIT_TAG, 'testnet')
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:71: Double quote to prevent globbing and word splitting

(shellcheck)